src/app/authentication/models/signUpModel.go

```diff
@@ -1 +1,14 @@
-package models
\ No newline at end of file
+package models
+
+type UserSignUp struct {
+	Name        string `json:"name" validate:"required"`
+	UserName    string `gorm:"column:username" json:"username" validate:"required"`
+	Email       string `json:"email" validate:"required,email"`
+	PhoneNumber uint64 `json:"phoneNumber" validate:"required,min=1000000000,max=9999999999,numeric"`
+	PanCard     string `json:"panCard" validate:"required"`
```

Comment:
I would suggest whoever implemented forgotPasswordModel should make this change so that he can update the same in his code and check its working correctly.